workers/unity/Packages/io.improbable.gdk.mobile/Editor/AndroidUtils.cs

```diff
@@ -194,5 +194,23 @@ private static bool TryGetApkPath(string rootPath, out string apkPath)
             apkPath = string.Empty;
             return false;
         }
+
+        private static bool UseEmbeddedSDK()
+        {
+            const string SDKPrefKey = "SdkUseEmbedded";
```

Comment:
could make this a static const string outside of the method?

Reply:
I personally prefer the local const if its only used here